### ISSUE TITLE
Poor F1 Score

### ISSUE THREAD

**Issue by SMDIndomitable** [huanidz/scaled-alpr-unconstrained#2]
Hi again, 

Sorry for the trouble. We have adjusted our dataset and trained as per your suggestion, however the f1 score is still only at 0.06 after 400 epochs. Do you mind sharing the dataset you trained with, so we can check with our dataset?

Thank you for your time and have a nice day!

**Comment by huanidz** [comment]
@SMDIndomitable 
Hi, for your concern, 

I found an error in my implementation related to loss functions and it cause the same problem on my dataset.

I will fix it and do a quick train on my dataset and push it to branch:main later. Will notice you when the patch is delivered (This should be quick).

Sorry for this problem, i haven't touch this for a long time and didn't notice the version at my center is differ to this git version.

**Comment by huanidz** [comment]
@SMDIndomitable 
By the way, i think since the last time you had asked, i did push some fix. Have you tried the latest version ?
For convenient, can you give me your chat connection ?

**Comment by SMDIndomitable** [comment]
May I know if your discord is "huanidz"?

I noticed the repo we are using currently is not the latest version. Should I update it after you fix the error or use the current version first? 

Thank you very much for quick response, appreciate it!

**Comment by huanidz** [comment]
@SMDIndomitable 
Yes it is: huanidz

Please try the latest version in the main branch. I think i did some loss fix earlier but not sure it solve your problem, so try it first.
